task_processing/plugins/smartstack.py

```diff
@@ -0,0 +1,34 @@
+import yaml
+
+
+YOCALHOST_IP = '169.254.255.254'
+TASKPROC_SERVICE_FILE = '/nail/etc/services/task_processing/smartstack.yaml'
```

Comment:
This is not a public interface of smartstack
 You get /nail/etc/services.yaml and /var/run/synapse/services/<json discovery file>